src/invoices/InvoiceDetails/InvoiceDetails.js

```diff
@@ -275,16 +276,13 @@ function InvoiceDetails({
         )}
         {
           isApproveConfirmationOpen && (
-            <ConfirmationModal
```

Comment:
are you deleting confirmation of approving invoice? I'm not sure this is a requirement

Reply:
No). I've created a new component `<ApproveConfirmationModal>` that includes information from `ConformationModal` and a new block with duplicate invoices if needed.

Reply:
but we still need this confirmation modal, maybe it makes sense to add new one - for duplicated invoices?

Reply:
Maybe I didn't catch your idea. As I remember it should be one Modal that asks 'Are you sure you want to approve invoice?' and below duplicate invoice list appears if such invoices exist.

Reply:
it's weird, I've asked dennis a question https://issues.folio.org/browse/UINV-192?focusedCommentId=94463&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-94463